safe_autonomy_sims/gym/inspection/weighted_inspection_v0.py

```diff
@@ -347,14 +347,11 @@ def _get_reward(self):
         )
 
         # Sparse rewards
-        reward += (
-            r.weighted_inspection_success_reward(
-                chief=self.chief, total_weight=self.success_threshold
-            )
-            if utils.closest_fft_distance(chief=self.chief, deputy=self.deputy)
-            < self.crash_radius
-            else -1.0
-        )
+        success_reward = r.weighted_inspection_success_reward(chief=self.chief, total_weight=self.success_threshold)
+        if success_reward > 0:
+            if utils.closest_fft_distance(chief=self.chief, deputy=self.deputy) < self.crash_radius:
```

Comment:
We can combine these two if statements

Reply:
See https://github.com/act3-ace/safe-autonomy-sims/pull/19#discussion_r1750848835

Reply:
Combined